### ISSUE TITLE
[Availability] <li> elements must be contained in a <ul> or <ol>

### ISSUE THREAD

**Issue by christinach** [pulibrary/allsearch_frontend#297]
### Steps to recreate

1. Go to https://allsearch.princeton.edu/?q=giraffe
2. Run axe dev tools
3. Note that you get `<li> elements must be contained in a <ul> or <ol>`

### URL

https://allsearch.princeton.edu/?q=giraffe

### Issues

List item does not have a <ul>, <ol> parent element

### Severity

Serious

### WCAG Conformance level (A, AA, AAA)

Level A

### User impact

For a list to be valid, it must have both parent and child elements. Parent elements can either be a set of ul tags or a set of ol tags. Child elements must be declared inside of these tags using the li tag.

Screen readers notify users when they come to a list, and tell them how many items are in a list. Announcing the number of items in a list and the current list item helps listeners know what they are listening to, and what to expect as they listen to it.

If you don't mark up a list using proper semantic markup in a hierarchy, list items cannot inform the listener that they are listening to a list when no parent is indicating the presence of a list and the type of list.

See [deque's page about this rule](https://dequeuniversity.com/rules/axe/4.8/listitem?application=AxeChrome)

### Recommended fix

Wrap <li> in a <ul>, <ol> parent element

### Acceptance criteria
- [ ] I can successfully use this feature with Voice Over in Safari.
- [ ] I have checked this feature using one of the following browser extensions and confirmed that it does not add any level A or level AA issues:
    * axe dev tools
    * Lighthouse
    * Wave
- [ ] ...
